lib/v4/migration-helpers/update-api-folder-structure.js

```diff
@@ -78,10 +80,13 @@ const updateApiFolderStructure = async (appPath) => {
   const apiDirs = await getDirsAtPath(apiDirCopyPath);
 
   for (const api of apiDirs) {
-    const apiName = normalizeName(api.name);
+    let apiName = pluralize.singular(normalizeName(api.name));
+    if (apiName !== api.name) {
```

Comment:
A bit picky but I find the `apiName` a little vague. Could you change it to something like `apiSingularName`

Also I think I introduced this `normalizeName` variable but it really serves no purpose. Would you mind removing it and just use ` _.kebabCase` directly?